integration_tests/pom.xml

```diff
@@ -28,14 +28,39 @@
     <artifactId>rapids-4-spark-integration-tests_2.12</artifactId>
     <version>0.2.0-SNAPSHOT</version>
 
+    <properties>
+        <slf4j.version>1.7.30</slf4j.version>
```

Comment:
Normally version dependency properties are placed in the top-level pom and used in the dependency management section to make sure everything's using a consistent version.  Is it important for it to be here instead?

Reply:
sure I can move, I was hoping to just remove or remove before PR but need to modify cudf and I haven't gotten back to getting it to build.  I'll move up.